### DISCUSSION TITLE
Randomized test sets from jrl-qp

### DISCUSSION THREAD

**Discussion by stephane-caron** [qpsolvers/qpbenchmark#36] [Ideas]
There are C++ problem generation routines in jrl-qp:

* https://github.com/jrl-umi3218/jrl-qp/blob/master/benchmarks/Solvers.cpp
* https://github.com/jrl-umi3218/jrl-qp/blob/master/include/jrl-qp/test/randomProblems.h

They could help improve the benchmark in qpsolvers.

**Comment by jcarpent** [comment]
You have complementary tools in ProxQP (https://github.com/Simple-Robotics/proxsuite/blob/a52c13cbe6286eebd2bc0157f7d901c3b5f681cf/include/proxsuite/proxqp/utils/random_qp_problems.hpp)
